### ISSUE TITLE
Disable MSHTML caching

### ISSUE THREAD

**Issue by guiscaranse** [r0x0r/pywebview#170]
I am with a big problem in my project, because I am current always trying stylesheets and javascript codes but sometimes things doesn't change because of the MSHTML cache, and I can't find anywhere to disable it caching for development purposes. It is really important for pywebview to let developers a option to disable cache, or even let it disabled by default.

**Comment by r0x0r** [comment]
Have you tried setting cache options in your HTML file?

**Comment by guiscaranse** [comment]
Yes, thats the workaround that I am currently doing right now. But take a look at chrome, on it's developers tools it is possible to disable cache temporarily or even forever. I don't know if more people had this issue too but it's something that should be considered

**Comment by shivaprsdv** [comment]
Yes, but the problem is we can't (for now) display the developer tools on Windows.
Another solution you might try is to enable the right-click context menu; there will
be an option to reload the page, and that would reload the cached resources.

**Comment by r0x0r** [comment]
IE DevTools is a no-go, as it is not a part of MSHTML. This seems to be the easiest approach to disable caching https://stackoverflow.com/questions/24347068/disable-webbrowser-caching
And another one https://msdn.microsoft.com/en-us/library/aa770041(v=VS.85).aspx#Controlling_Download_and_Execution, but to be honest I have no idea how to implement this.

One of the approaches described in the first link could be attached to the debug=True parameter. @guiscaranse would you be able to give it a go?

**Comment by guiscaranse** [comment]
`@shivaprsdv: Another solution you might try is to enable the right-click context menu; there will
be an option to reload the page, and that would reload the cached resources.`

Oh, that's amazing to hear! I've enabled that and it seens working fine right now.

`@r0x0r: One of the approaches described in the first link could be attached to the debug=True parameter. @guiscaranse would you be able to give it a go?`

Yes, I would love to test it. I am not quite familiar with vb.net and the winforms bridge to python otherwise I would love to help write something to it.

**Comment by shivaprsdv** [comment]
@r0x0r Looks like we can add `IsWebBrowserContextMenuEnabled` to the `debug` flag.

**Comment by shivaprsdv** [comment]
Implemented the above WinForms feature in 62c3818fc799859fa3fcd42d4edbaa173daa81b0.

**Comment by shivaprsdv** [comment]
@guiscaranse Can we consider this as solved?
@r0x0r We could reopen this if we ever decide to disable caching via code.
